crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/controller/ContributionController.java

```diff
@@ -88,4 +92,27 @@ public ResponseEntity<Void> requestTransfer(@Valid @RequestBody ApiContributionT
         return ResponseEntity.ok().build();
     }
 
+
+    @PostMapping(value = "/check-contribution-rule", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(description = "Check if contribution Rule is applicable")
+    @ApiResponse(responseCode = "200",
+            content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE
+            )
+    )
+    @DefaultHTTPErrorResponse
+    public ResponseEntity<Boolean> checkContributionRule(
+            @Parameter(description = "Data required to check contribution rule",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
+                            schema = @Schema(implementation = ApiCalculateContributionRequest.class)
+                    )
+            )
+            @Valid @RequestBody
+            ApiMaatCalculateContributionRequest maatCalculateContributionRequest) {
```

Comment:
There are quite a few mandatory fields in this request object, such as the `RepId`, `applicationId` etc. that I don't really think we should be passing? So imo we'd be better off just creating a new schema for this endpoint.